src/coreclr/jit/compiler.h

```diff
@@ -3084,6 +3084,15 @@ class Compiler
         return (callMethodHandle == impInlineRoot()->info.compMethodHnd);
     }
 
+    NamedIntrinsic gtGetNamedIntrinsicForCall(GenTreeCall* call)
+    {
+        if (call->gtCallMoreFlags & GTF_CALL_M_SPECIAL_INTRINSIC)
+        {
+            return lookupNamedIntrinsic(call->gtCallMethHnd);
+        }
+        return NI_Illegal;
+    }
```

Comment:
I added it because `lookupNamedIntrinsic` is protected + it allows me to omit `GTF_CALL_M_SPECIAL_INTRINSIC` flag every time.

Reply:
I've removed it